deepchem/models/losses.py

```diff
@@ -909,6 +909,69 @@ def loss(atom_vocab_task_atom_pred: torch.Tensor,
         return loss
 
 
+class EdgePredictionLoss(Loss):
+    """
+    Unsupervised graph edge prediction loss.
```

Comment:
Can you add some more detail about what this loss computes?

Reply:
Done